docs/README.md

```diff
@@ -1,5 +1,13 @@
 # Kueue documentation
 
+Welcome to Kueue - A Kubernetes native way for queueing workloads
```

Comment:
way?

Perhaps system? Or you can be more explicit saying "APIs and controller"

Reply:
And using the word "job" will be more familiar for most people before they read "concepts"